arlo_server/models.py

```diff
@@ -56,6 +58,8 @@ class Election(BaseModel):
     )
     jurisdictions_file = relationship("File")
 
+    __table_args__ = (db.UniqueConstraint("audit_name", "organization_id"),)
```

Comment:
💯

Reply:
FYI it's better practice to do `(organization_id, audit_name)`, as that index can be used to select all audits by organization, whereas the other order doesn't let you do that.

Reply:
Oh right duh! Can you tell I've been out of the game for a year?